### PR TITLE
Theonewhostands/alc 11

### LINKED ISSUE #11: Dockerize Alice
Spinning up Alice as fast as possible and as cleanly as possible is of high priority. Dockerizing her will enable a smoother flow of execution and development.

### PR DESCRIPTION
**Pull request by QriusDev**

### CODE REVIEW

File: src/commands/Command.bus.ts [hunk 1]
```diff
@@ -36,13 +36,53 @@ class CommandBus {
 
 const commandBus = new CommandBus();
 
+/**
+ * Command: commands
+ * Usage:
+ * Permissions: All
+ */
 commandBus.addHiddenCommand('commands', new ListCommand);
```
Comment (ALostNPC) [inline src/commands/Command.bus.ts:44]: Permissions states all, but is added as hidden command

File: src/commands/Command.bus.ts [hunk 2]
```diff
@@ -36,13 +36,53 @@ class CommandBus {
...
+ * Permissions: All
+ */
 commandBus.addPublicCommand('promote', new PromoteCommand);
+
+/**
+ * Command: contest
+ * Usage:
+ * Permissions: Admin
+ */
 commandBus.addPublicCommand('contest', new ContestCommand);
```
Comment (ALostNPC) [inline src/commands/Command.bus.ts]: Permission states Admin, but set as public command

File: src/commands/Command.bus.ts (hunk 1 above)
Comment (QriusDev) [reply src/commands/Command.bus.ts:44]: the visibility doesn't mean its for admins. The visibility is soley for the list-command to know what commands it should show vs those it shouldn't.

File: src/commands/Command.bus.ts (hunk 2 above)
Comment (QriusDev) [reply src/commands/Command.bus.ts]: true. but you put that there lol. I'll fix it though.

*edit: by "put that there", i mean, its an admin command and you wrote it to be public.

**Review by ALostNPC — APPROVED** [review]

**Pull request merged**